src/i18n/en.js

```diff
@@ -183,6 +183,21 @@ You can avoid saturating your Database and possible removals. Do you need help?
     testAb: 'A/B Test Campaigns',
     title: 'Exclusive Features',
   },
+  faq: {
+    answer_1: `In your account, select Update Plan. Choose your Plan and fill your billing info. Click on "I have a promotion code", then enter your code and validate it.`,
+    answer_2: `Due to your amount of Contacts, you can acquire a Monthly High Volume Plan that will suit your business needs with a cheaper cost per Email.`,
+    answer_3: `Prepaid Plans' Credits don't have any type of expiration. You can use them for your sendings whenever you want. Also, Credits do accumulate.`,
+    answer_4: `You can pay using International credit cards from Mastercard, VISA or Amex. In Mexico, Argentina or Spain, bank transfers are allowed for purchases from US$77.`,
+    answer_5: `There is no minimum time to stay using Doppler. If you downgrade, only the current month will be charged. Need a contract? we can do it, but we wont ask for one.`,
+    answer_6: `If you choose a Pay in advance Plan, after meeting the agreed time, it will renew automatically for the same period you've previously selected.`,
+    question_1: `How to apply a Promotion Code to access a discount?`,
+    question_2: `Is there a Monthly Plan for more than 100,000 Contacts?`,
+    question_3: `Do Prepaid Plans' Credits have a deadline for their use?`,
+    question_4: `Which payment methods are available to purchase a Paid Plan?`,
+    question_5: `Is there a minimum contract of permanence when buying a Paid Plan?`,
+    question_6: `How does renewal works when paying 3, 6 or 12 months in advanced?`,
+    title: `Frequently ask questions`,
+  },
```

Comment:
Is it possible to use an array to avoid the need of declaring a constant (it is error-prone when you add a new QA you have to touch two files)?

```suggestion
  faq: {
    qas: [
     {
       answer: `In your account, select Update Plan. Choose your Plan and fill your billing info. Click on "I have a promotion code", then enter your code and validate it.`,
       question: `How to apply a Promotion Code to access a discount?`,
     },
     {
       answer: `Due to your amount of Contacts, you can acquire a Monthly High Volume Plan that will suit your business needs with a cheaper cost per Email.`,
       question: `Is there a Monthly Plan for more than 100,000 Contacts?`,
     },
     {
       answer: `Prepaid Plans' Credits don't have any type of expiration. You can use them for your sendings whenever you want. Also, Credits do accumulate.`,
       question: `Do Prepaid Plans' Credits have a deadline for their use?`,
     },
     {
       answer: `You can pay using International credit cards from Mastercard, VISA or Amex. In Mexico, Argentina or Spain, bank transfers are allowed for purchases from US$77.`,
       question: `Which payment methods are available to purchase a Paid Plan?`,
     },
     {
       answer: `There is no minimum time to stay using Doppler. If you downgrade, only the current month will be charged. Need a contract? we can do it, but we wont ask for one.`,
       question: `Is there a minimum contract of permanence when buying a Paid Plan?`,
     },
     {
       answer: `If you choose a Pay in advance Plan, after meeting the agreed time, it will renew automatically for the same period you've previously selected.`,
       question: `How does renewal works when paying 3, 6 or 12 months in advanced?`,
     },
    ],
    title: `Frequently ask questions`,
  },
```

Reply:
it's better!

Reply:
Yep, but I do not know if it is possible, it could require changes, maybe you should need to edit the test of the language files, etc.

Reply:
It should be possible to do, I already investigated about it

Reply:
I couldn't get a default implementation that has react-intl for this. What it is is that several people do it in the same way as it is in the PR

An example: https://codesandbox.io/s/qzplkqz9kj?file=/src/index.js